src/glmtools.jl

```diff
@@ -222,22 +222,22 @@ end
 linkfun(::IdentityLink, μ::Real) = μ
 linkinv(::IdentityLink, η::Real) = η
 mueta(::IdentityLink, η::Real) = one(η)
-inverselink(::IdentityLink, η::Real) = η, one(η), oftype(η, NaN)
+inverselink(::IdentityLink, η::Real) = η, one(η), convert(float(typeof(η)), NaN)
 
 linkfun(::InverseLink, μ::Real) = inv(μ)
 linkinv(::InverseLink, η::Real) = inv(η)
 mueta(::InverseLink, η::Real) = -inv(abs2(η))
 function inverselink(::InverseLink, η::Real)
     μ = inv(η)
-    return μ, -abs2(μ), oftype(μ, NaN)
+    return μ, -abs2(μ), convert(float(typeof(μ)), NaN)
```

Comment:
These are likely not needed as `μ = inv(η)` will (almost?) always be a float. Though I guess that doesn't hurt much.